README.rst

```diff
@@ -27,15 +27,25 @@ Features
 ========
 
 * Highly available master nodes (ASG) behind ELB
-* Worker Auto Scaling Group
+* Worker Auto Scaling Group with node pools support
 * Flannel overlay networking
-* Cluster autoscaling (using kube-aws-autoscaler_)
+* Cluster autoscaling (using cluster-autoscaler_)
+* Kubernetes DNS with node-local dnsmasq as daemonset and CoreDNS resolver for cluster.local domain.
 * Route53 DNS integration via `External DNS`_
 * AWS IAM integration via kube2iam_
 * Standard components are installed: kube-dns, heapster, dashboard, node exporter, kube-state-metrics
-* Webhook authentication and authorization (roles "ReadOnly", "PowerUser", "Administrator")
+* Webhook authentication and authorization (roles "ReadOnly", "ProductionDeployer", "Manual", "Emergency", "Administrator")
```

Comment:
`PowerUser` correct, and `ProductionDeployer` should not be listed there. `Production Deployer` just maps to `ReadOnly` in terms of cluster access.

Reply:
I updated it.